### ISSUE TITLE
watch invocation should wrap makefile path

### ISSUE THREAD

**Issue by sagiegurari** [sagiegurari/cargo-make#445]
seed-rs/seed#515

**Comment by Type1J** [comment]
@sagiegurari Thanks for looking at this issue!

**Comment by sagiegurari** [comment]
@Type1J i have a development branch 0.32.2 which might have a fix for this.
can you maybe test it as well on your use case?

**Comment by Type1J** [comment]
I checked it out, did a
```bash
cargo uninstall cargo-make
git checkout 0.32.2
cargo install --path .
cd <place with Seed project>
cargo make start
```
It looks like that works. Thanks!

**Comment by sagiegurari** [comment]
great. thanks a lot for checking.
i have few more unrelated changes before i release a new official version.

**Comment by sagiegurari** [comment]
this is now released in the new cargo-make version 0.32.2
